### PR TITLE
Adds db tests and few fixes

### PR DESCRIPTION
**Pull request by sulochan**

### CODE REVIEW

**Comment by sulochan** [comment]
WIP .. (need to complete, region and hosts) but good for some initial review.

File: craton/inventory/tests/unit/db/test_cells.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+from craton.inventory.tests.unit.db import base
+
+
+cell1 = {'region_id': 1, 'project_id': 1, 'name': 'cell1'}
+
+
+class CellsDBTestCase(base.DBTestCase):
+
+    def test_cells_create(self):
+        dbapi.cells_create(self.context, cell1)
```
Comment (sigmavirus24) [inline craton/inventory/tests/unit/db/test_cells.py]: Is there an assertion to be made here?

File: test-requirements.txt [hunk 2]
```diff
@@ -5,8 +5,10 @@
...
 flask
 flask-restful
+fixtures>=1.3.1,<2.0  # Apache-2.0/BSD
 jsonschema
 nose
 mock
-oslotest>=1.10.0 #Apache-2.0
+oslotest>=1.10.0 # Apache-2.0
 testtools>=1.4.0
+taskflow>=1.26.0 # Apache-2.0
```
Comment (sigmavirus24) [inline test-requirements.txt]: This seems unrelated.

**Comment by jimbaker** [comment]
Important change to get in, and looking good except for the minors that Ian pointed out.

**Comment by sigmavirus24** [comment]
With the exception that Travis is angry, I'm in favor of this. Thanks for cleaning up that test.
